core/src/Streamly/Internal/Data/Stream/StreamK.hs

```diff
@@ -49,8 +49,7 @@
 module Streamly.Internal.Data.Stream.StreamK
     (
     -- * The stream type
-      Stream(..) -- XXX stop exporting this
-    , StreamK
+      StreamK(..) -- XXX stop exporting this
```

Comment:
The comment is wrong.

Also keep exporting "Stream" as well for backward compatibility.